include/z5/attributes.hxx

```diff
@@ -1,17 +1,30 @@
 #pragma once
 
+#include <fstream>
 #include "nlohmann/json.hpp"
 #include "z5/handle/handle.hxx"
 
-namespace fs = boost::filesystem;
+#ifdef WITH_BOOST_FS
+    namespace fs = boost::filesystem;
+#else
+    #if __GCC__ > 7
+        namespace fs = std::filesystem;
+    #else
+        namespace fs = std::experimental::filesystem;
+    #endif
+#endif
```

Comment:
Maybe it makes sense to pull this out into some utility header and include the header everywhere. WDYT?

Reply:
Definitely. Thought about that too while refactoring, but wanted to test it first and then forgot about it.